orchagent/portsorch.cpp

```diff
@@ -2563,13 +2566,22 @@ void PortsOrch::doLagTask(Consumer &consumer)
                 }
                 else if (fvField(i) == "oper_status")
                 {
-                    if (fvValue(i) == "down")
+                    operation_status = fvValue(i);
+                    if (!string_oper_status.count(operation_status))
                     {
-                        gNeighOrch->ifChangeInformNextHop(alias, false);
+                        SWSS_LOG_ERROR("Invalid operation status value:%s", operation_status.c_str());
+                        it++;
+                        continue;
                     }
-                    else
+
+                    gNeighOrch->ifChangeInformNextHop(alias,
+                                                      (operation_status == "down") ?
+                                                      false : true);
```

Comment:
Is it the same as `operation_status != "down"` ?

Reply:
Yes.

Reply:
Could you simplify the code as I mentioned?

---
In reply to: [433554505](https://github.com/Azure/sonic-swss/pull/1242#discussion_r433554505) [](ancestors = 433554505)